concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -892,7 +892,7 @@ Hale-1973-1798-890	Food_and_Clothing-87	plug a leak	v	07b1.37
 Hale-1973-1798-891	Food_and_Clothing-88	waste	v	07b1.38	79	WASTE
 Hale-1973-1798-892	Food_and_Clothing-89	economize	v	07b1.39		
 Hale-1973-1798-893	Food_and_Clothing-90	save (keep unused)	v	07b1.40	1997	PRESERVE
-Hale-1973-1798-894	Food_and_Clothing-91	beat foods	v	07b1.41		
+Hale-1973-1798-894	Food_and_Clothing-91	beat foods	v	07b1.41	3768	POUND (FOOD)
```

Comment:
Hm, that would be two POUND (FOOD) in the same list.
I would tend to unmap that latter one as it might rather be about beating e.g. cream and not about pre-processing a grain or other starch-source as in the glosses that were mapped to BEAT.
But I'm not sure.